modules/nw-udn-best-practices.adoc

```diff
@@ -0,0 +1,31 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="considerations-for-udn_{context}"]
+= Best practices for UserDefinedNetwork
```

Comment:
I think it should be noted, as best practice, using NAD should be minimal, only for cases the UDN CRD doesnt cover.

Reply:
done